spec/requests/admin/users/users_export_spec.rb

```diff
@@ -31,10 +31,13 @@
   end
 
   it "shows the correct fields", :aggregate_failures do
-    expect(response.body).to include('Admin1,admin1,admin1@gmail.com,Good Standing,"06 May, 2020","06 May, 2020",[]')
+    admin_registered_at = admin.registered_at.strftime("%d %b, %Y")
+    user_registered_at = user.registered_at.strftime("%d %b, %Y")
```

Comment:
This may be a bit too verbose but I want to test if it works and then I can improve on it. In addition, I'm changing the way local dates work in this view in an upcoming PR.